docs/admin/authorization.md

```diff
@@ -79,18 +81,28 @@ The file format is [one JSON object per line](http://jsonlines.org/).  There sho
 one map per line.
 
 Each line is a "policy object".  A policy object is a map with the following properties:
-  - `user`, type string; the user-string from `--token-auth-file`. If you specify `user`, it must match the username of the authenticated user.
-  - `group`, type string; if you specify `group`, it must match one of the groups of the authenticated user.
-  - `readonly`, type boolean, when true, means that the policy only applies to GET
-      operations.
-  - `resource`, type string; a resource from an URL, such as `pods`.
-  - `namespace`, type string; a namespace string.
+  - Versioning properties:
+    - `apiVersion`, type string; valid values are "abac/v1". Allows versioning and conversion of the policy format.
+    - `kind`, type string: valid values are "Policy". Allows versioning and conversion of the policy format.
+
+  - Subject-matching properties:
+    - `user`, type string; the user-string from `--token-auth-file`. If you specify `user`, it must match the username of the authenticated user. `*` matches all requests.
+    - `group`, type string; if you specify `group`, it must match one of the groups of the authenticated user. `*` matches all requests.
+
+  - `readonly`, type boolean, when true, means that the policy only applies to get, list, and watch operations.
+
+  - Resource-matching properties:
+    - `resourceGroup`, type string; an API group, such as `extensions`. `*` matches all API groups.
+    - `resource`, type string; a resource, such as `pods`. `*` matches all resource requests.
+    - `namespace`, type string; a namespace string. `*` matches all resource requests.
+
+  - Non-resource-matching properties:
+  - `nonResourcePath`, type string; matches the non-resource request paths (like `/version` and `/apis`). `*` matches all non-resource requests. `/foo/*` matches `/foo/` and all of its subpaths.
```

Comment:
For API resources, can we call this `apiGroup`, and imply the /api or /apis prefix?  For miscellaneous resources, keep `nonResourcePath`, and forbid specifying both `apiGroup` and `nonResourcePath`?